index.html

```diff
@@ -7,6 +7,34 @@
     <link rel="shortcut icon" type="image/jpg" href="assets/images/rotten_trailers_favicon.png"/>
     <title>Rotten Trailers</title>
 </head>
+<header>
+    <div id=logo>
+        <img src="Rottentrailerslogo will go here" alt="Rottern Trailers Logo"/>
```

Comment:
I think the src should be changed to since it is a local path href: "assets/images/rotten_trailers_logo-01.png"